source/features/split-issue-pr-search-results.tsx

```diff
@@ -52,8 +52,10 @@ function init() {
 		issueLink.classList.remove('selected');
 		title.textContent = title.textContent.replace('issue', 'pull request');
 	} else if (/\btype=Issues\b/.test(location.search) && !/\bis:issue\b/.test(getSearchQuery())) {
+		title.textContent = title.textContent
+			.replace(/issue\b/, 'issue or pull request')
+			.replace(/issues/, 'issues and pull requests');
```

Comment:
Test: https://github.com/search?q=alsdka&type=Issues

Reply:
`'issues and/or pull requests'` ...

Reply:
lol the link was supposed to only show one result, but pasting the link here created a second result. Check out now

Reply:
Oh lol, I was actually suggesting an sentence change. That [previous link](https://github.com/search?q=alsdka&type=Issues) showed two results, both of them in pull requests. That's why I suggested to reword it to `'issues and/or pull requests'`, because there were zero (0) issues.

Reply:
Nah I think it’s ok with “and” since the search contains both types. Also easier to read